src/overlays/actors/ovl_En_Niw/z_en_niw.c

```diff
@@ -21,52 +40,1012 @@ const ActorInit En_Niw_InitVars = {
     (ActorFunc)EnNiw_Update,
     (ActorFunc)EnNiw_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Niw_0x80891060/EnNiw_Init.asm")
+static f32 D_80893484[] = { 5000.0f, -5000.0f,};
+static f32 D_80893486[] = { 5000.0f, 3000.0f, 4000.0f,};
+
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_HIT5, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_NO_PUSH | OC1_TYPE_ALL, OC2_TYPE_2, COLSHAPE_CYLINDER, },
+    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+    { 15, 25, 4, { 0, 0, 0 } },
+};
+
+static Vec3f D_808934C4 = { 90000.0f, 90000.0f, 90000.0f, };
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_U8(targetMode, 6, ICHAIN_CONTINUE),
+    ICHAIN_F32_DIV1000(gravity, -2000, ICHAIN_CONTINUE),
+    ICHAIN_F32(targetArrowOffset, 0, ICHAIN_STOP),
+};
+
+static Vec3f D_808934DC = { 90000.0f, 90000.0f, 90000.0f, };
+
+static Vec3f D_808934E8 = { 90000.0f, 90000.0f, 90000.0f, };
+
+static s32 pad = 0;
+
+void EnNiw_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnNiw* this = (EnNiw*)thisx;
+    Vec3f D_Temp = D_808934C4;
```

Comment:
This is probably not actually real anyways, the data should just be put directly in the function

Reply:
![static included](https://user-images.githubusercontent.com/42048411/123522748-30e81a80-d674-11eb-89bc-0485b1e56b4d.png)
doesn't look like it can be placed in the function, the asm wants to copy it locally from out reference

Reply:
Okay, so I didn't try this before because its silly, but 
```c
    static Vec3f D_808934E8 = { 90000.0f, 90000.0f, 90000.0f, };
    Vec3f tempVec3f;
    ...   
    tempVec3f = D_808934E8;
    ...
    Math_Vec3f_Copy(&this->unk2BC, &tempVec3f);
```
This is equal to the other best match (data outside of function, copy to local, pass as addr)... but, why?
Why would someone intentionally put data directly into their function, and then copy it, and then pass the address of the copy to Vec3fCopy (copy again to struct)?

If the data was outside of the function I could see some half asleep dev copying the Vec3f to the function by mistake (forgetting to make it a pointer), but this makes no sense.

Reply:
Maybe try with
```c
    //static Vec3f D_808934E8 = { 90000.0f, 90000.0f, 90000.0f, };
    Vec3f tempVec3f = { 90000.0f, 90000.0f, 90000.0f, };
    ...   
    //tempVec3f = D_808934E8;
    ...
    Math_Vec3f_Copy(&this->unk2BC, &tempVec3f);
```